x-pack/platform/plugins/shared/alerting/common/saved_objects/rules/mappings.ts

```diff
@@ -10,6 +10,19 @@ import type { SavedObjectsTypeMappingDefinition } from '@kbn/core/server';
 export const alertMappings: SavedObjectsTypeMappingDefinition = {
   dynamic: false,
   properties: {
+    artifacts: {
+      type: 'object',
```

Comment:
nit: `type: 'object'` can be omitted. The `properties` should be enough. Same for `investigation_guide`.

Reply:
See 14ce40db2f7bd187ae78bfaa2b7cfd32010258bf.